vim/core/oni-plugin-git/src/vcs.ts

```diff
@@ -42,6 +63,7 @@ export interface VersionControlProvider {
     canHandleWorkspace(dir?: string): Promise<boolean>
     getStatus(): Promise<StatusResult | void>
     getRoot(): Promise<string | void>
+    getBlame(args: BlameArgs): Promise<Blame>
```

Comment:
Cool! I like that this is abstracted out and that the layer itself doesn't need to know about Git - just about the `Blame` / `BlameArgs`.